SpatialGDK/Documentation/content/directory-structure.md

```diff
@@ -1,22 +1,19 @@
 <%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
```

Comment:
I think we need to address this around the repo so it doesn't look so janky. Perhaps in a clean-up PR later. @ElleEss thoughts?

Reply:
Your call - depends on how much you want to make people aware of any instabilities or anything likely to change.
For Unity, the consensus was to leave this on every page but amend to "alpha". 
TBH as we are now publishing docs on the website, it's probs overkill to have this at the top of every page.